packages/sdk-node/src/authenticateRequest.ts

```diff
@@ -102,7 +89,7 @@ export const decorateResponseWithObservabilityHeaders = (res: ServerResponse, re
 const isDevelopmentFromApiKey = (apiKey: string): boolean =>
   apiKey.startsWith('test_') || apiKey.startsWith('sk_test_');
 
-const getRequestUrl = (req: IncomingMessage): URL => {
+export const getRequestUrl = (req: IncomingMessage): URL => {
```

Comment:
🙃 please check if this also gets exported from the `sdk-node` . If it's get exported I would advise to move this to utils and import it here.

Reply:
@dimkl Thanks for mentioning it, but I think we don't have any issues with exports.